pom.xml

```diff
@@ -73,6 +73,17 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>2.1.214</version>
+        </dependency>
+        <dependency>
+            <groupId>com.j256.ormlite</groupId>
+            <artifactId>ormlite-jdbc</artifactId>
+            <version>6.1</version>
+        </dependency>
+
```

Comment:
Once your previous PR is merged, you won't need to add these dependencies again, so after that is merged, please do a rebase on origin/main

Reply:
done